src/main/resources/TestNg.xml

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
+
+<suite name="Tests suite">
```

Comment:
The name of the file and suite name could be more descriptive. SeleniumTestSuite.xml or something like that sounds more clear rather than mysterious TestNG.xml file :)

Reply:
I will change it in the newer PR when I open one. I am merging this one :)